### PR TITLE
Add the ability to switch the dashboard visibility and style after initial setup

### PR DESCRIPTION
**Pull request by idanboa**
usage:
`LifetimeTracker.configure(updateClosure: LifetimeTrackerDashboardIntegration(visibility: .alwaysHidden, style: .circular).refreshUI)
`

### CODE REVIEW

**Comment by idanboa** [comment]
Hi, this is following our email correspondence :)
I want to add the ability to change the visibility of the dashboard on the fly after the initial setup,
so that i can view what components are currently live in the app and not only for monitoring leaks.

File: Sources/LifetimeTracker.swift [hunk 1]
```diff
@@ -210,7 +210,14 @@ public extension LifetimeTrackable {
         instance = LifetimeTracker(onLeakDetected: onLeakDetected, onUpdate: onUpdate)
     }
     
-    private let onUpdate: UpdateClosure
+    /// Set a new update closure, use to set the visibility/style of the dashboard.
```
Comment (freak4pc) [inline Sources/LifetimeTracker.swift:213]: Another option here could be just

```swift
    public var onUpdate: UpdateClosure {
        didSet { onUpdate?(trackedGroups) }
    }
```

Comment (idanboa) [reply Sources/LifetimeTracker.swift:213]: Thanks, i'll give it a go asap 👍

Comment (krzysztofzablocki) [reply Sources/LifetimeTracker.swift:213]: any updates @idanboa ?

Comment (idanboa) [reply Sources/LifetimeTracker.swift:213]: pushed changes

**Review by krzysztofzablocki — APPROVED** [review]

**Pull request merged**